test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6701,11 +6701,56 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
                     sname = 'default'
                 else:
                     sname = id
+
                 # Default max cycle length is 1200 seconds (20m)
                 self.expect(SCHED, {'state': 'scheduling'}, op=NE, id=sname,
-                            interval=1, max_attempts=1200)
+                            interval=1, max_attempts=1200,
+                            trigger_sched_cycle=False)
         return rc
 
+    def run_sched_cycle(self, sched="default"):
+        """
+        Convenience method to start and finish a sched cycle
+
+        :param sched - the scheduler to kick a cycle for
+        :type sched - str
+
+        :return tuple of (start time, end time) of the new sched cycle
+        """
+        old_val = self.status(SCHED, 'scheduling', id=sched)
```

Comment:
Consider adding '[0]['scheduling']' to the end.  This way old_val will be the 'True' or 'False' you need to use to set it back.